### DISCUSSION TITLE
vscode remote ssh bash scripting

### DISCUSSION THREAD

**Discussion by roshanroy1805** [microsoft/vscode-discussions#1543] [Extension Development QnA] [answered]
I've recently started using Visual Studio Code and am exploring the Remote SSH option. I have a Linux VM and want to use VS Code to manage bash scripts. I successfully connected to the VM and was able to run the program in the VS Code terminal without any issues. However, when I try to execute the program by clicking the play button in VS Code, it shows the output but doesn't read my input when prompted. Here's a screenshot of the problem I'm encountering.

![Screenshot 2024-08-28 at 9 07 39 AM](https://github.com/user-attachments/assets/27d01c44-0ef0-4e05-8a6a-26e31fa77984)

I have tried below on my Linux vm apt install direnv echo 'eval "$(direnv hook bash)"' >> ~/.bashrc echo 'eval "$(direnv hook zsh)"' >> ~/.zshrc source ~/.bashrc source ~/.zshrc

On VScode, I have tried to set "direnv.path.executable": "/usr/bin/direnv"

But still the same

Thanks in advance

**Comment by gjsjohnmurray** [comment, marked as answer]
Unless you are an extension developer this is not the place for your question. Please see pinned topic https://github.com/microsoft/vscode-discussions/discussions/5
